firmwares/zooz/ZEN76-V01.json

```diff
@@ -13,6 +13,12 @@
 		}
 	],
 	"upgrades": [
+		{
+			"version": "10.30.1",
+			"changelog": "* Improved Direct Association Group 2 for the basic set on/off functionality.",
```

Comment:
As of right now, the change log entry for 10.30.1 is missing for the ZEN76: https://www.support.getzooz.com/kb/article/1151-zen76-s2-on-off-switch-change-log/

I have asked Zooz support to add the missing entry.

However, I am pretty sure it's identical to 10.30.1 for the ZEN71 since they are similar devices and otherwise have similar firmware updates, so I copied the entry from that one for now.

Reply:
Zooz has now added the change log entry for 10.03 for the ZEN76. The wording I assumed before matches exactly what they've posted.